packages/routing/cardstack/searcher.js

```diff
@@ -34,11 +65,11 @@ module.exports = class SpacesSearcher {
         attributes: { 'url-segment': `/${cardType}/${cardId}` },
         relationships: {
           'primary-card': {
-            data: { type: cardType, id: cardId }
+            data: { type: card.type, id: card.id }
```

Comment:
My one remaining concern is here.

At this point, we just went through the trouble of loading `card`. And it may have a ton of `included` cards already embedded inside it, if we've done a good job optimizing the index.

But we throw away everything other than `type` and `id`, which will force the `DocumentContext` to reload our card and any other includes.

I think we can build a better response here that `includes` the `card` as well as anything that was included inside it.

Reply:
ah, i see what you mean. cool thanks for the feedback.